confluent_kafka/src/confluent_kafka.c

```diff
@@ -1508,52 +1581,38 @@ rd_kafka_conf_t *common_conf_setup (rd_kafka_type_t ktype,
 	conf = rd_kafka_conf_new();
 
         /*
-         * Default config (overridable by user)
+         * Set debug contexts first to capture all events including plugin loading
          */
+         if ((vo = PyDict_GetItemString(confdict, "debug"))) {
+                if (common_conf_set_special(confdict, conf, "debug", vo)) {
+                        goto outer_err;
+                }
+         }
 
-        /* Enable valid offsets in delivery reports */
-        rd_kafka_conf_set(conf, "produce.offset.report", "true", NULL, 0);
+        /*
+         * Default config (overridable by user)
```

Comment:
This comment doesn't seem to be true anymore.

Reply:
Although green this is not actually part of the changes. I think it may be leftover from when we consolidated the topic configuration into the global one. 

https://github.com/confluentinc/confluent-kafka-python/blob/master/confluent_kafka/src/confluent_kafka.c#L1510-L1512

I can go ahead and remove it here though